lib/stdlib/test/digraph_utils_SUITE.erl

```diff
@@ -248,6 +248,14 @@ tree(Config) when is_list(Config) ->
 
     ok.
 
+%% OTP-9040
+traversals(Config) when is_list(Config) ->
+    G = digraph:new([]),
+    add_edges(G, [{a,b},{b,c},{c,d},{d,e}]),
+    [a,b,c,d,e] = digraph_utils:preorder(G),
+    [e,d,c,b,a] = digraph_utils:postorder(G),
+    ok.
```

Comment:
There is no test for an empty digraph.

```
1> digraph_utils:preorder(digraph:new()).
** exception error: no match of right hand side value {#{},[]}
     in function  digraph_utils:preorder/1 (digraph_utils.erl, line 394)
```

Maybe also test some more complicated graphs. For example:

```suggestion
    add_edges(G, [{0,1},{1,2},{2,0}]),
    [a,b,c,d,e,0,1,2] = digraph_utils:preorder(G),
    [e,d,c,b,a,2,1,0] = digraph_utils:postorder(G),

    add_edges(G, [{x,0},{y,1},{z,2}]),
    [z,2,0,1,y,x,a,b,c,d,e] = digraph_utils:preorder(G),
    [1,0,2,z,y,x,e,d,c,b,a] = digraph_utils:postorder(G),

    ok.
```